docs/index.md

```diff
@@ -167,6 +167,8 @@ Now you can access the server at `http://localhost:8080`.
 - 7-Tage-Inzidenzen der letzten Tage [https://corona-germany.justus-d.de/](https://corona-germany.justus-d.de/)
 - Die Krankenhausampel für einen (beliebigen) Landkreis als freies und anpassbares Skript für alle Webseitenbetreiber [https://krankenhausampel.info/](https://krankenhausampel.info/)
 - Corona_Ampel_Bayern, Übersicht über aktuelle Zahlen für Bayern und deren Landkreise. [https://corona-ampel-bayern.de](https://corona-ampel-bayern.de)
+- Casumer Corona Tracker - Eine Svelte PWA zur Überwachung frei konfigurierbarer Landkreise (https://cct.greeninc.ga)
```

Comment:
Please add `[https://https://cct.greeninc.ga]` befor `(https://cct.greeninc.ga)`
resulting in `- Casumer Corona Tracker - Eine Svelte PWA zur Überwachung frei konfigurierbarer Landkreise [https://https://cct.greeninc.ga](https://cct.greeninc.ga)`

Reply:
already done by me